mousecursor.go

```diff
@@ -0,0 +1,61 @@
+package flutter
+
+import (
+	"errors"
+	"fmt"
+	"github.com/go-flutter-desktop/go-flutter/plugin"
+	"github.com/go-gl/glfw/v3.3/glfw"
+)
+
+const mousecursorChannelName = "flutter/mousecursor"
+
+// mousecursorPlugin implements flutter.Plugin and handles method calls to the
+// flutter/mousecursor channel.
+type mousecursorPlugin struct {
+	channel    *plugin.MethodChannel
```

Comment:
Sorry, didn't catch this in the first review.
p.channel can also be removed from the plugin struct.